packages/hardhat/.eslintrc.json

```diff
@@ -1,6 +1,4 @@
 {
   "root": true,
-  "parser": "@typescript-eslint/parser",
-  "plugins": ["@typescript-eslint"],
-  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "prettier"]
+  "extends": ["../../.eslintrc.json", "prettier"]
```

Comment:
If we put `"prettier"` into the root `eslintrc`, we can completely remove this file. Note that `"prettier"` is also in `extends` in the `frontend` package.

Reply:
i'm not sure. somewhere i read that `prettier` must be the last item in that array to work properly. but don't remember where. @raulrpearson can you confirm or should i have we have another look?

Reply:
found it. it's written here: https://paulintrognon.fr/blog/typescript-prettier-eslint-next-js. but it's not official documentation.

Reply:
yes, the prettier should typically be last because it actually disables a bunch of lint rules (that other configs might have activated) that become irrelevant when prettier is taking care of the formatting.

BTW, I think the similarities between configs will start to melt away once we add linting and formatting of solidity code.

Reply:
https://prettier.io/docs/en/integrating-with-linters.html
https://github.com/prettier/eslint-config-prettier#installation

Reply:
yep. i agree with raul. moved eslint files back to package-level in [dc6246d](https://github.com/yieldgate/yieldgate/pull/29/commits/dc6246d77ed27d3878781edcc5b556a4a8a5360f).